node-red/node-red-setup.sh

```diff
@@ -13,9 +13,9 @@ if [ ! -f /service/node-red-init.flag ]; then
     {
         "id": "e9738faf9b0558bb",
         "type": "mqtt-broker",
-        "name": "thin-edge-setup.io",
-        "broker": "thin-edge-setup.io",
-        "port": "8883",
+        "name": "tedge",
```

Comment:
Internal services can be referenced by their service name, there is no need to use the `container_name` here.